src/rpc/getAddress.ts

```diff
@@ -0,0 +1,9 @@
+import {Wallet} from "../interfaces";
+
+export async function getAddress(wallet: Wallet): Promise<string> {
+  const state = wallet.getPluginState();
```

Comment:
create some utility function getKeyPair which checks plugin state and generates keys if there isn't anything in state

Reply:
Use that method in every rpc that requires keys